nservicebus/nhibernate/accessing-data_customizing-session_nhibernate_[6,7).partial.md

```diff
@@ -0,0 +1,7 @@
+## Customizing the session
+
+Versions 5 and below supported customizing the instantiation of the `ISession`. This is done by hooking up to the creation process by providing a custom delegate:
```

Comment:
I mean, the whole page is for NHibernate 6 and 7 (core 5 and 6). I am a bit confused by this para. The partial name suggest it applies to NH 6 (core 5). The para says that customizing session was allowed in 5 and below. 

When splitting into partials we usually remove any version specifications from the narrative so I would remove that "Versions 5 and below..." and create a partial for NH7/Core 6 which says "not available any more"